data-tool/flows/config.py

```diff
@@ -109,6 +123,12 @@ class _Config():  # pylint: disable=too-few-public-methods
     ACCOUNT_SVC_CLIENT_SECRET = os.getenv('ACCOUNT_SVC_CLIENT_SECRET')
     ACCOUNT_SVC_TIMEOUT = os.getenv('ACCOUNT_SVC_TIMEOUT')
 
+    # batch delete flow
+    BATCHES = int(os.getenv('BATCHES', sys.maxsize))
+    BATCH_SIZE = int(os.getenv('BATCH_SIZE', 300))
```

Comment:
I think if there are no values given, we should default these two to 0. 

Anyone who runs this flow needs to set the values explicitly.

Think you'll need to also update the logic to maybe exit out of the flow immediately when one of these values are 0 as you'll run into division by zero errors.

Reply:
I introduced `BATHES` because sometimes we may want to run specific number of runs (1 or 2 used for debugging etc.)...most of time, the value will be set dynamically via calculation. And it is used as 
```python
batches = min(math.ceil(total/batch_size), config.BATCHES)
```
in the [code](https://github.com/bcgov/lear/blob/4184b4dc53b5fdece1572dff535ce256c7165da7/data-tool/flows/batch_delete_flow.py#L405). Do you still want me to set it to 0 by default?

But I'll handle the divison by zero error, which I didn't consider well before.

Reply:
I'm thinking we should still set `BATCHES` to 0 by default just to be careful.

We can maybe add some validation to the start of `batch_delete_flow` and exit right away if there are zero values.

```python
if config.BATCH_SIZE <= 0:
    raise ValueError("BATCH_SIZE must be explicitly set to a positive number")
if config.BATCHES <= 0:
    raise ValueError("BATCHES must be explicitly set to a positive number")
```